### DISCUSSION TITLE
localization

### DISCUSSION THREAD

**Discussion by karakaptan7** [laurens94/vue-timeline-chart#5] [Q&A] [answered]
awesome component  🤌🏻👍🏻

is there a localization option?

**Comment by laurens94** [comment, marked as answer]
Thank you! 🙏 

At the moment there's no localization option, but I'm open to any suggestions of improving the way timestamps are labeled right now.

There is an option to overwrite the timestamp labeling function. It's not pretty, but the current implementation is replaceable by providing the prop `renderTimestampLabel` with a function that expects the following parameters:

`function(timestamp: number, scale: { unit: string, step: number})`

You can have a look at the internal function here:

https://github.com/laurens94/vue-timeline-chart/blob/603be860e7ddfae0c474c446b6868b2289d45255/src/components/Timeline.vue#L186:L204

Let me know in case you need some help with it!
